engine/classes/ElggEntity.php

```diff
@@ -1686,7 +1686,14 @@ protected function update() {
 		unset($persisted_entity);
 
 		if ($allow_edit) {
-			$allow_edit = _elgg_services()->events->trigger('update', $this->type, $this);
+			$allow_edit = elgg_trigger_before_event('update', $this->type, $this);
```

Comment:
I vaguely remember a good reason I didn't migrate entity "update" to before/after events, and I don't recall what it was!